### PR TITLE
Update argument name for bind_rows and set operations.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Argument name update

### .id

Change it to `id_column_nam`

### first_id

Change it to `current_df_name`

### ignore_column_data_type

Change it to `force_data_type`

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/util.R [hunk 1]
```diff
@@ -1475,39 +1475,49 @@ one_hot <- function(df, key) {
...
 
 #'Wrapper function for dplyr::bind_rows to support named data frames when it's called inside dplyr chain.
 #'@export
-bind_rows <- function(..., .id = NULL, first_id = '', ignore_column_data_type = FALSE) {
+bind_rows <- function(..., id_column_name = NULL, current_df_name = '', force_data_type = FALSE) {
+  # get a list of argument names to resolve data frame names passed to this bind_rows.
+  # only exception is the current data frame which is passed via dplyr pipe operation (%>%).
+  # it becomes period (.) instead of actual df name.
+  args <- extract_argument_names(...)
+  browser()
```
Comment (hideaki) [inline R/util.R]: Remove browser()

File: R/util.R [hunk 2]
```diff
@@ -1475,39 +1475,49 @@ one_hot <- function(df, key) {
...
   # If the dplyr::bind_rows is called within a dplyr chain like df1 %>% dplyr::bind_rows(list(df_2 = df2, df_3 = df3), .id="id"),
   # since df1 does not have a name, the "id" column of the resulting data frame does not have the data frame name for rows from df1.
   # To workaround this issue, set a name to the first data frame with the value specified by fistLabel argument as a pre-process
   # then pass the updated list to dplyr::bind_rows.
   dataframes_updated <- list()
   # with dplyr:::flatten_bindable API, create a list of data frames from arguments passed to bind_rows.
   dataframes <- dplyr:::flatten_bindable(rlang::dots_values(...))
-  if(ignore_column_data_type | stringr::str_length(first_id) >0) {
+  if(force_data_type | stringr::str_length(current_df_name) >0) {
```
Comment (hideaki) [inline R/util.R]: Use || instead of | in if statement condition to make sure the resulting logical vector is of length 1.

File: R/util.R [hunk 3]
```diff
@@ -1475,39 +1475,49 @@ one_hot <- function(df, key) {
...
     if(!is.null(names(dataframes))) {
       # iterate data frames list by name as a key.
       for (name in names(dataframes)) {
         # for the first item, it's the data frame passed via %>% operator, so it does not have a key (data frame name) yet.
         # so populate the key with the value passed by first_id argument.
-        if(stringr::str_length(first_id) > 0 & index == 1) {
-          # if ignore_column_data_type is set, force character as column data types
-          if(ignore_column_data_type) {
-            dataframes_updated[[first_id]] <- dplyr::mutate_all(dataframes[[1]], funs(as.character))
+        if(stringr::str_length(current_df_name) > 0 & index == 1) {
```
Comment (hideaki) [inline R/util.R]: Use && instead of & in if statement condition to make sure the resulting logical vector is of length 1.

File: R/util.R (hunk 2 above)
Comment (hidekoji) [reply R/util.R]: Fixed as suggested.

File: R/util.R (hunk 3 above)
Comment (hidekoji) [reply R/util.R]: Fixed as suggested

File: R/util.R (hunk 1 above)
Comment (hidekoji) [reply R/util.R]: removed browser()

**Pull request merged**